.github/workflows/main.yml

```diff
@@ -48,4 +48,9 @@ jobs:
         run: |
           python -m pytest --cov=agents/ --cov=gym_xiangqi/ --cov=examples/ --cov-fail-under=25
 
-
+      # Runs unit tests for Xiangqi environment components
+      - name: Run unit tests for Xiangqi environment components
```

Comment:
Oh is this needed? Those tests should already be running under the build CI here: https://github.com/tanliyon/gym-xiangqi/pull/14/checks?check_run_id=2000275603.

Reply:
Oh oops. I did not know if it was running all the tests. I will remember to remove those lines with another PR.